solutions/security/detect-and-alert/suppress-detection-alerts.md

```diff
@@ -134,13 +130,28 @@ With alert suppression, detection alerts aren’t created for the grouped source
 
     :::{image} /solutions/images/security-timeline-button.png
     :alt: Investigate in timeline button
-    :width: 350px
+    :width: 250px
     :screenshot:
     :::
 
 * Alert details flyout — Select **Take action** → **Investigate in timeline**.
 
 
+## Impact of closing suppression alerts [security-alert-suppression-impact-close-alerts]
+
+As a best practice, avoid closing suppresssion alerts (alerts generated for alert suppression) before an active suppression window ends. Closing alerts early can interrupt alert suppression or cause unexpected changes. 
+
+For example, say you set the suppression time period to 5 minutes and specify to group alerts by the `host.name` field. The first time an event meets the rule's criteria, an alert is created. Over the next 5 minutes, any subsequent duplicate events are suppressed and grouped by unique `host.name` value. If you close that first alert before the active suppression window ends (the 5 minute suppression time period), alert suppression ends and restarts when the next qualifying event meets the suppression criteria. 
+
+
+:::{image} /solutions/images/security-alert-suppression-close-alert-example.png
+:alt: Example of suppression configuration for a rule
+:screenshot:
+:width: 450px
+:::
+
+{applies_to}`stack: ga 9.2` To ensure suppression continues after closing suppression alerts, you can set the `securitySolution:suppressionBehaviorOnAlertClosure` [advanced setting](/solutions/security/get-started/configure-advanced-settings.md#suppression-window-behavior) to **Continue until supppression window ends**. The default selection is **Restart suppression**, which enforces the behavior described above.
```

Comment:
I'd phrase this something like: "You can change the default behavior to continue suppressing alerts till the end of suppression window after you closed an investigated alert. This will prevent restarting the suppression window every time you close an alert. To do so change the settings...."